### DISCUSSION TITLE
11.0 Roadmap?

### DISCUSSION THREAD

**Discussion by amwx** [AvaloniaUI/Avalonia#7983] [General]
For transparency, can we get an updated roadmap/target feature list that is planned for the 11.0 release. 
I know there's a couple issues related to 11.0/the major release and things to expect but they've kinda fallen out of date: #2239, #3538

Some of the features have been completed (like merging the core libraries, UIA, better mobile support, drag/drop etc), or closed (like replacing ItemsPresenter with ItemsRepeater), but is everything on those lists still planned? Are there things being pushed back to 12.0 or later? New Renderer? Styling system refactor? Changes to public API (like 'xxImpl')? etc...

This doesn't need to be a comprehensive list, but it'd be nice to know the plan for some of the bigger features planned that have been talked about for a couple years now and where they are in the release cycle moving forward.

**Comment by grokys** [comment]
Hi @amwx - we're tracking progress on 11.0 on this board currently: https://github.com/AvaloniaUI/Avalonia/projects/10
